src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -701,7 +701,7 @@ TEST_F(HMICapabilitiesTest, Comparison_ccpuVersion) {
   std::shared_ptr<HMICapabilitiesForTesting> hmi_capabilities =
```

Comment:
Consider use auto instead of repeated using <HMICapabilitiesForTesting>

Reply:
@ychernysheva thank you, I want to leave this variant

Reply:
@ydementieiev I would recommend to follow @ychernysheva advise. It is enough info in 
`std::make_shared<HMICapabilitiesForTesting>` to understand what type is returned. 
But the statement is more clear and readable. Compare: 

```cpp

std::shared_ptr<HMICapabilitiesForTesting> hmi_capabilities = 
    std::make_shared<HMICapabilitiesForTesting>(mock_app_mngr);
```

and 

```cpp
auto hmi_capabilities = std::make_shared<HMICapabilitiesForTesting>(mock_app_mngr);
```